english/scarab/goddess.md

```diff
@@ -16,4 +16,6 @@ Do you:
 
 [scream?](../blah-balh/blah-blah.md)
 
-[ask for forgiveness?](https://www.youtube.com/watch?v=fGuR4GOV5L0)
\ No newline at end of file
+[ask for forgiveness?](https://www.youtube.com/watch?v=fGuR4GOV5L0)
+
+[Stack rocks until the next move comes to you?](english/scarab/stack-rocks/stack-rocks.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.